apps/schedules/test/hours_of_operation_test.exs

```diff
@@ -17,8 +17,8 @@ defmodule Schedules.HoursOfOperationTest do
       assert %Departures{} = week_1
       assert %Departures{} = saturday_0
       assert %Departures{} = saturday_1
-      assert %Departures{} = sunday_0
-      assert %Departures{} = sunday_1
+      assert :no_service = sunday_0
```

Comment:
This is unrelated to my code, but this test started failling. I don't really understand the implications of this, maybe the service on this route changed?

Reply:
We can disregard this, I've noticed this recently failing on other branches as well and I haven't gotten around to investigating yet.

Reply:
Gotcha, so just take out these changes then?

Reply:
Yes please 😅